torchaudio/backend/utils.py

```diff
@@ -53,6 +53,9 @@ def set_audio_backend(backend: Optional[str]):
     elif backend == 'sox_io':
         module = sox_io_backend
     elif backend == 'soundfile':
+        warnings.warn(
+            '"soundfile" backend is planned to change the function signatures in 0.9.0. '
+            'Please refer to https://github.com/pytorch/audio/issues/903')
```

Comment:
When I see a warning like this, I think "Ok, how can I change my code to migrate and suppress the warning?". This warning may give the impression soundfile should not be used, which is not the case.

If we want to warn, then I'd say we should have a mechanism for the user to use the new interface right away.

Reply:
(offline comment on [internal document](https://docs.google.com/document/d/1ekaLxE9BzSlZ3JcTZD90icfvtO3N6JKs5XmBuZigcDI/edit#))